contentcuration/contentcuration/frontend/channelEdit/views/CurrentTopicView.vue

```diff
@@ -397,6 +398,15 @@
             actionCallback: () => changeTracker.revert(),
           });
         });
+
+        // If clipboardNode comes back undefined, the clipboard hasn't been
+        // loaded yet - so we must load it before returning the above Promise.
+        const clipboardNode = this.getContentNode(window.user.clipboard_root_id);
+        if (!clipboardNode) {
+          return this.loadClipboardTree().then(() => copyPromise);
+        } else {
+          return copyPromise;
+        }
```

Comment:
Perhaps we should add this to the clipboard `copyAll` vuex action? I believe there other locations where this could happen.

Reply:
Good thinking - updated accordingly